TranslationBackend/tests/Feature/InitialDataTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\DB;
+use Tests\TestCase;
+use Throwable;
+
+class InitialDataTest extends TestCase
+{
+    use DatabaseMigrations;
+    /**
+     * A basic feature test example.
```

Comment:
Might need to change this comment to better suit ur test case

Reply:
That's true, I've fixed it now.